.rubocop_todo.yml

```diff
@@ -312,6 +312,7 @@ Layout/LineLength:
     - 'app/models/red_line_boundary_change_validation_request.rb'
     - 'app/views/api/v1/validation_requests/index.json.jbuilder'
     - 'spec/factories/planning_application.rb'
+    - 'spec/helpers/validation_request_helper_spec.rb'
```

Comment:
can't this be amended instead?

Reply:
It's a string `url = "<a href=\"/planning_applications/#{planning_application.id}/documents/#{document.id}/edit\">proposed-floorplan.png</a>"`
I could just wrap this line in a rubocop:disable ?

Reply:
or use some string formatting method https://ruby-doc.org/core-3.0.3/String.html#method-i-25, but yeah I'm always uncomfortable with shelving a rule for a whole file because of a single offense... Up to you, not critical